src/docs/030_news/2024/3.3-release.adoc

```diff
@@ -0,0 +1,57 @@
+:filename: 030_news/2024/3.3-release.adoc
+:jbake-title: Release v3.3
+:jbake-date: 2024-02-18
+:jbake-type: post
+:jbake-tags: docToolchain
+:jbake-status: published
+:jbake-menu: news
+:jbake-author: Ralf D. Müller
+:icons: font
+
+ifndef::imagesdir[:imagesdir: ../../../images]
+
+== docToolchain release v3.3 (Bugfix-Release)
+
+
+=== About This Release
+
+This release fixes a problem with the docker image and dependencies in a restricted environment.
+The docker image should already contain all required dependencies and thus should be able to run in an enviroment without internet connection (build pipeline).
+With the latest im provements, a change was introduced which broke this feature.
+This resulted in timeouts for the dependency resolution.
+
+We now managed to fix this again.
+
+We also introduced an enviroment variable DTC_DOCKER_PREFIX to specify an internal docker hub for the docker image.
+This should also help to run docToolchain in restricted environments.
+
+Other changes fix problems with custom ruby extensiosn for PDF generation and features for `publishToConfluence`
+
+IMPORTANT: this release breaks the `generateDeck` task.
```

Comment:
i could not verify that 3.3 will break this task...

Reply:
ok. I will remove it

Reply:
done